code/game/objects/items/weapons/tools/misc.dm

```diff
@@ -52,6 +52,36 @@
 	max_upgrades = 1
 	spawn_tags = null
 
+/obj/item/tool/multitool_improvised
+	name= "improvised multitool implant"
+	desc = "A jury-rigged implant, holding cobbled-together tools. For those who are more interested in tool carrying than scared of tetanus."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "multitool_improvised"
+	force = WEAPON_FORCE_WEAK
```

Comment:
That's really weak.
```suggestion
	force = WEAPON_FORCE_PAINFUL
```

Reply:
Ah I had it as weak initially to match the damage of the others, but then it is housed in a toolbox so can see why more damage would make sense